Assets/Mapzen/TileBounds.cs

```diff
@@ -0,0 +1,56 @@
+using System;
+using System.Collections.Generic;
+
+namespace Mapzen
+{
+    [System.Serializable]
+    public struct TileArea
+    {
+        public LngLat min;
+        public LngLat max;
+        public int zoom;
+
+        public TileArea(LngLat min, LngLat max, int zoom)
+        {
+            this.min = min;
+            this.max = max;
+            this.zoom = zoom;
+        }
+    }
+
+    public class TileBounds
+    {
+        public readonly TileAddress min;
+        public readonly TileAddress max;
+
+        private TileArea area;
+
+        public TileBounds(TileArea area)
+        {
+            this.area = area;
+
+            min = TileAddress.FromLngLat(area.min, area.zoom);
```

Comment:
Yes I agree. I was looking at enforcing the usage through `Debug.Assert`, but those seems to be a no-op in Unity. Since this is part of the Mapzen SDK we couldn't use anything that Unity provides for that (https://blogs.unity3d.com/2015/08/25/the-unity-assertion-library/), so we might have to use exceptions.